infrastructure/zk/src/contract.ts

```diff
@@ -129,12 +129,19 @@ export async function deployL2ThroughL1({ includePaymaster }: { includePaymaster
     let l2DeployLog = fs.readFileSync('deployL2.log').toString();
     const l2DeploymentEnvVars = [
         'CONTRACTS_L2_SHARED_BRIDGE_ADDR',
+        'CONTRACTS_L2_ERC20_BRIDGE_ADDR',
         'CONTRACTS_L2_TESTNET_PAYMASTER_ADDR',
         'CONTRACTS_L2_WETH_TOKEN_IMPL_ADDR',
         'CONTRACTS_L2_WETH_TOKEN_PROXY_ADDR',
         'CONTRACTS_L2_DEFAULT_UPGRADE_ADDR'
     ];
     updateContractsEnv(`etc/env/l2-inits/${process.env.ZKSYNC_ENV!}.init.env`, l2DeployLog, l2DeploymentEnvVars);
+    // erc20 bridge is now deployed as shared bridge, but we still need the config var:
```

Comment:
why do we need it? it is optional in the server

Reply:
We need it to test the old SDK,